NickvisionMoney.GNOME/Blueprints/account_view.blp

```diff
@@ -265,117 +265,192 @@ Adw.Bin _root {
+            hexpand: true;
+            vexpand: true;
             
-            Adw.PreferencesGroup _transactionsGroup {
-              title: _("Transactions");
-              margin-top: 7;
+            Adw.PreferencesGroup _visualizeGroup {
+              title: _("Visualize");
+              margin-top: 4;
               margin-start: 10;
               margin-end: 10;
               header-suffix: Gtk.Box {
                 orientation: horizontal;
-                spacing: 6;
                 
-                Gtk.DropDown _sortTransactionByDropDown {}
+                Gtk.Button _graphBackButton {
+                  valign: center;
+                  icon-name: "go-previous-symbolic";
+                  tooltip-text: _("Back");
+                  
+                  styles ["flat"]
+                }
                 
-                Gtk.Box {
-                  orientation: horizontal;
+                Gtk.Button _graphNextButton {
                   valign: center;
+                  icon-name: "go-next-symbolic";
+                  tooltip-text: _("Next");
                   
-                  Gtk.ToggleButton _sortFirstToLastButton {
-                    icon-name: "view-sort-descending-symbolic";
-                    tooltip-text: _("Sort From First To Last");
-                    active: bind _sortLastToFirstButton.active inverted bidirectional;
+                  styles ["flat"]
+                }
+              };
+              
+              Gtk.Box {
+                orientation: vertical;
+                spacing: 6;
+                
+                Adw.Carousel _carousel {
+                  Gtk.DrawingArea _incomeExpenseOverTimeImage {
+                    height-request: 300;
+                    hexpand: true;
                   }
                   
-                  Gtk.ToggleButton _sortLastToFirstButton {
-                    icon-name: "view-sort-ascending-symbolic";
-                    tooltip-text: _("Sort From Last To First");
+                  Gtk.DrawingArea _incomeExpensePieImage {
+                    height-request: 200;
+                    hexpand: true;
                   }
                   
-                  styles ["linked"]
+                  Gtk.DrawingArea _incomeExpensePerGroupImage {
+                    height-request: 300;
+                    hexpand: true;
+                  }
+                  
+                  Gtk.Box {
+                    orientation: horizontal;
+                    
+                    Gtk.DrawingArea _incomeByGroupImage {
+                      height-request: 300;
+                      hexpand: true;
+                    }
+                    
+                    Gtk.Separator {
+                      orientation: vertical;
+                    }
+                    
+                    Gtk.DrawingArea _expenseByGroupImage {
+                      height-request: 300;
+                      hexpand: true;
+                    }
```

Comment:
I think this can be simplified, we can have one DrawingArea and draw 2 graphs with `width / 2` and x offset for the second one, I'm gonna try...

Reply:
> I'm gonna try...

Ok :)